specification/cdn/resource-manager/Microsoft.Cdn/preview/2019-06-15-preview/examples/WafListPolicies.json

```diff
@@ -108,7 +108,7 @@
                   }
                 ]
               },
-              "cdnEndpointLinks": [
+              "endpointLinks": [
                 {
```

Comment:
Please update allt he samples with other fields as well .. TransformType and ActionType

Reply:
I've updated the examples with the appropriate transforms fields.